crates/bevy_reflect/src/lib.rs

```diff
@@ -1,6 +1,4 @@
-// FIXME(15321): solve CI failures, then replace with `#![expect()]`.
-#![allow(missing_docs, reason = "Not all docs are written yet, see #3492.")]
-// `rustdoc_internals` is needed for `#[doc(fake_variadics)]`
+#![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]
 #![allow(internal_features)]
```

Comment:
Why did you remove this? It might have warned about being unused because it is only used with that feature enabled. Maybe put this under a cfg_attr instead?

Reply:
I see! Is https://github.com/bevyengine/bevy/pull/16561/commits/229ef7f5317b8a44c8019798b8a5620ff128c27a what you were thinking?